frontend/src/components/activePage/TableInstance/TableInstance.tsx

```diff
@@ -58,6 +64,30 @@ const TableInstance: FC<ITableInstanceProps> = ({ ...props }) => {
     });
   };
 
+  const checked = !!instances.filter(i => selectiveDestroy?.includes(i.id))
```

Comment:
Would extract `instances.filter(i => selectiveDestroy?.includes(i.id))` into a variable since it's used more than once and it's quite verbose...